packages/angular/src/migrations/update-13-2-0/update-testing-imports.ts

```diff
@@ -39,7 +39,7 @@ export default async function (tree: Tree) {
       tree,
       {},
       {
-        'jasmine-marbles': '~0.8.3',
+        'jasmine-marbles': '~0.9.1',
```

Comment:
jasmine-marbles should only be bumped to 0.9.1 for workspaces with RxJS 7. Since Angular doesn't force this, the migration should check the version of RxJS 7 before bumping from 0.8.x to 0.9.x.